zip-####.rst

```diff
@@ -155,62 +207,77 @@ SHOULD be applied to the Zcash Testnet for testing purposes.
 Dev Fund allocation
 -------------------
 
-Starting at the first Zcash halving in 2020, until the second halving in 2024,
-20% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
-consists of the following three slices:
+Starting at the second Zcash halving in 2024, until the third halving in 2028,
+15% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
+consists of the following allocations:
 
-* 35% for the Bootstrap Project (denoted **BP slice**);
-* 25% for the Zcash Foundation, for general use (denoted **ZF slice**);
-* 40% for additional "Major Grants" for large-scale long-term projects
-  (denoted **MG slice**).
+* 5% for the Zcash Foundation (for internal work and grants);
+* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 
+and contingent upon an ecosystem mandated "non-direct funding model" for the future;
+* 4% for Zcash Community Grants for continuation of their current activities as-is;
+* 2% for Qedit in support of their ZSA activities, and other Zcash protocol support;
 
-The slices are described in more detail below. The fund flow will be implemented
+Following the first year, when ECC will no longer receive their 4% allocation, 
+those block subsidies will be distributed as 1% to ZCG, 1% to ZecHub, 1% to FPF,
+and 1% to ZF.
+
+This proposal mandates the ecosystem to design and deploy a "non-direct funding model"
+as generally recommended in Josh Swihart's proposal.
+<https://forum.zcashcommunity.com/t/zcash-funding-bloc-a-dev-fund-proposal-from-josh-at-ecc/47187>
+
+"Dev Fund" block subsidies will be administered through two organizations:
+1. Zcash Foundation  (for ECC, ZCG)
+2. Financial Privacy Fund (for Qedit, ZecHub)
+
+The allocations are described in more detail below. The fund flow will be implemented
 at the consensus-rule layer, by sending the corresponding ZEC to the designated
-address(es) for each block. This Dev Fund will end at the second halving (unless
+address(es) for each block. This Dev Fund will end at the third halving (unless
 extended/modified by a future ZIP).
 
 
-BP slice (Bootstrap Project)
+BP allocation (Bootstrap Project)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This slice of the Dev Fund will flow as charitable contributions from
+These funds SHALL be received and administered by ZF.
```

Comment:
Is the inclusion of the Bootstrap portion within the ZF's administration something that both Bootstrap and ZF have agreed upon in principle? If yes, this needs a reference to that agreement (given that this is a significant change from ZIP 1014 and ZIP 214). If not, then this would need to be removed (and similarly elsewhere in the ZIP draft).

@daira also makes the point that this increases the risk of key compromise that ZF would need to manage.

Reply:
At lines 112-117,  I intend to acknowledge the general intention of how direct block subsidies would be administered, and that the details have not been formally agreed upon by parties (presumably) involved (these being ZF, ECC, ZCG, FPF, Qedit, ZecHub).

Given that my ZIP seems to have a minority of support, I don't think it is critical to speculate/ jump into details of how/ who would be administering which block subsidies, or who would receive directly vs. indirectly (hypothetically speaking, if my ZIP gained popularity and were enacted,  the ECC and Qedit would be welcome to change their current opinions, and opt to receive funds directly).

ZF is currently holding the risk of compromise of their and ZCGs funds.  Risk of key compromise is decreases with perpetually devalued ZEC, so by coincidence the ZF and ECC currently have that advantage.  I would certainly entertain the idea of digging into maximizing ZEC administrator opsec if that is (or becomes) a predominant ecosystem concern.